src/components/Button/Button.css

```diff
@@ -126,5 +126,6 @@
 }
 
 .rc-tooltip {
+  max-width: 200px;
```

Comment:
maybe also center? but that's just personal preference

Reply:
I tried but to get it centered perfectly is not a trivial task for `rc-tooltip`. It seems like you have to get the width of the button it is enclosing with and calculate offsets from there.

Maybe it will be easier if we use the `Tooltip` component from `antd` in the future

Reply:
🤯 ok that's fine then